ci-operator/config/kcp-dev/controller-runtime-example/kcp-dev-controller-runtime-example-main.yaml

```diff
@@ -0,0 +1,20 @@
+binary_build_commands: make build
+build_root:
+  from_repository: true
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e
+  commands: make test-e2e
```

Comment:
odd that this works here directly, vs what we have to do in kcp (tests.step.tests.as)?
```
tests:
- as: lint
  steps:
    test:
    - as: lint
      commands: make lint
```

Reply:
it's the old-style (what we have here) vs new-style (elsewhere) approach. surprised that the generator gives you this by default

Reply:
https://issues.redhat.com/browse/DPTP-3073